.gitignore

```diff
@@ -0,0 +1,125 @@
+### Python template
```

Comment:
I added a `.gitignore` file generated by the Python template (I saw there is a simple Python script in this repo for CI flow).